### ISSUE TITLE
Backdoc-generated documentation is used in a commercial product

### ISSUE THREAD

**Issue by kkomelin** [chibisov/backdoc#9]
Hi Gennady,

I'm happy to inform you that Backdoc is used for generating documentation for [Showerma Drupal module](http://codecanyon.net/item/showerma-presentation-engine-for-drupal/13403848?ref=dexteny).

Congrats with having early commercial adopters! ;)

Best,
Konstantin

**Comment by chibisov** [comment]
Cool! But I can't see any docs there. Are they bundled in module?

**Comment by kkomelin** [comment]
Yes, they are. I've not yet decided whether I want to open them or not. 
However, if you're interested to take a look at the docs I can send you them through email.
